central/main.go

```diff
@@ -818,6 +823,7 @@ func debugRoutes() []routes.CustomRoute {
 			Authorizer:    user.WithRole(role.Admin),
 			ServerHandler: h,
 			Compression:   true,
+			EnableAudit:   true,
```

Comment:
Is it possible to modify state of ACS using these endpoints?

Reply:
I have seen "POST" request handling there. That's why I have added it to the list. But after looking a bit deeper, it does not persist anything. I'll remove it.